### PR TITLE
Generate sourcemaps from loop-breaker

### PR DESCRIPTION
**Pull request by wylieconlon**
```javascript
import loopBreaker from 'loop-breaker';

const source = 'const myCode = "hello world";';
const result = loopBreaker(source);

const transformedCode = result.code;
const sourceMap = result.map;
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "loop-breaker",
   "version": "0.1.0",
   "description": "Transforms JavaScript code to ensure that loops do not run indefinitely",
-  "main": "lib/index.js",
```
Comment (wylieconlon) [inline package.json]: @outoftime This file didn't seem to exist

File: src/index.js [hunk 2]
```diff
@@ -65,5 +68,8 @@ export default function(str) {
       this.traverse(loop);
     }
   });
-  return recast.print(ast).code;
+  const result = recast.print(ast, {
+    sourceMapName: 'map.json',
```
Comment (outoftime) [inline src/index.js]: Should we follow the convention of just adding `.map` to the filename?

Comment (wylieconlon) [reply src/index.js]: Good idea.

**Comment by wylieconlon** [comment]
@outoftime Please take another look. Updating the map name, which doesn't affect the output.

**Review by outoftime — APPROVED** [review]

**Pull request merged**